<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnTech::GuiQt::ProjectSettings::EditorWidget</class>
 <widget class="QMainWindow" name="UnTech::GuiQt::ProjectSettings::EditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>480</height>
   </rect>
  </property>
  <widget class="PropertyListView" name="propertyView"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PropertyListView</class>
   <extends>QTreeView</extends>
   <header>gui-qt/common/properties/propertylistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
